doc/man3/EVP_MAC.pod

```diff
@@ -354,6 +357,7 @@ EVP_MAC_do_all_provided() returns nothing at all.
       const char *cipher = getenv("MY_MAC_CIPHER");
       const char *digest = getenv("MY_MAC_DIGEST");
       const char *key = getenv("MY_KEY");
+      size_t keylen = strlen(key);
```

Comment:
not a good pattern to do strlen(key) and then check key == NULL further down..

Reply:
Good point.
Fixed.